.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     parallelism: 1
     docker:
       - image: 'cypress/included:3.4.0'
-    working_directory: ~/wheel
+    working_directory: ~/neetoui-challenge
```

Comment:
I saw the same changes made in https://github.com/shamilsdq/neetoui-challenge/pull/8#discussion_r788547941. Why are same changes repeated in each PR?

Reply:
Hi, sorry about that. I had created branch for 2nd issue from the branch for 1st issue.